docs/TabbedForm.md

```diff
@@ -824,55 +824,54 @@ Check [the `<AutoSave>` component](./AutoSave.md) documentation for more details
 
 ## Role-Based Access Control (RBAC)
 
-Fine-grained permissions control can be added by using the [`<TabbedForm>`](./AuthRBAC.md#tabbedform) and [`<FormTab>`](./AuthRBAC.md#tabbedform) components provided by the `@react-admin/ra-rbac` package. 
+Fine-grained permissions control can be added by using the [`<TabbedForm>`](./AuthRBAC.md#tabbedform) component provided by the `@react-admin/ra-rbac` package.
+
+Use in conjunction with ra-rbac's [`<TabbedForm.Tab>`](#tabbedformtab-1) and add a `name` prop to the `Tab` to define the resource on which the user needs to have the 'write' permissions for.
```

Comment:
Unless I'm mistaken this anchor (`#tabbedformtab-1`) now points to nothing, so you need to remove it (same in the other file btw)